src/Util/Reflection.php

```diff
@@ -37,6 +37,8 @@ public function getProperty($methodName)
         if (preg_match('/^('.$pattern.')(.+)$/i', $methodName, $matches)) {
             return $matches[2];
         }
+        
+        return null;
```

Comment:
The point of CS is consistency :) And so that we don't have to keep arguing over it. There'll be no end to it :stuck_out_tongue: